frontend/src/metabase/entities/containers/EntityForm.jsx

```diff
@@ -8,12 +8,28 @@ import ModalContent from "metabase/components/ModalContent";
 
 import entityType from "./EntityType";
 
+import type { Entity } from "metabase/lib/entities";
+
+export function getForm(entityDef: Entity, formName?: any = null) {
+  // 1. named form
+  // 2. default `form`
+  // 3. first of the named `forms`
+  return formName
+    ? // $FlowFixMe
+      entityDef.forms[formName]
+    : entityDef.form
+    ? entityDef.form
+    : Object.values(entityDef.forms)[0];
```

Comment:
The third option feels iffy to me. I'm not sure if `Object.values(...)[0]` is guaranteed to return the same value each time. Probably not a real issue if in practice ordering is defined, but it feels weird to rely on ordering.

If we do want to have some form be the default, maybe the entity def should do that manually?

```js
const FORMS = {someForm: {}, otherForm: {}}

const MyEntity = createEntity({
  forms: FORMS,
  form: FORMS.otherForm
})
```

Reply:
Looks like my info is outdated and insertion order is preserved: https://stackoverflow.com/a/23202095/607070

Also, I see this behavior is tested below 👍